### PR TITLE
Added codebook card

### PR DESCRIPTION
**Pull request by alvaro-at-xanadu**
Changing the PennyLane Carnival card for a link to the Xanadu Quantum Codebook

### CODE REVIEW

File: jinja/index.html [hunk 1]
```diff
@@ -57,17 +57,17 @@ <h5>Demos <i class="fas fa-angle-double-right"></i></h5>
...
                             <div style="width: 100%; min-height: 198px;">
                                 <h3 class="card-title"></i>
-                                    <strong>Celebrate</strong>
+                                    <strong>Code</strong>
                                 </h3>
-                                <!-- <img src="img/carnival_logo_splash2.png" style="margin-top: 40px;" width="200px"/> -->
-                                <p class="mb-1 black-text px-3">Learn about the PennyLane Quantum Carnival, which took place November 2021.</p>
-                                <div class="white-text d-flex justify-content-center" style="margin-top: 24px;">
-                                    <h5>Take a look <i class="fas fa-angle-double-right"></i></h5>
+                                <!-- <img src="img/codebook_blur.png" style="margin-top: 40px;" width="200px"/> -->
```
Comment (co9olguy) [inline jinja/index.html:67]: anything in `<!-- ... -->` is an HTML comment (ignored by browser), just for future reference

File: jinja/index.html [hunk 2]
```diff
@@ -57,17 +57,17 @@ <h5>Demos <i class="fas fa-angle-double-right"></i></h5>
...
+                                <p class="mb-1 black-text px-3"> Use PennyLane to explore various quantum computing topics in the new Xanadu Quantum Codebook</p>
```
Comment (co9olguy) [inline jinja/index.html:68]:
```suggestion
                                <p class="mb-1 black-text px-3"> Use PennyLane to explore various quantum computing topics in the Quantum Codebook</p>
```

**Review by co9olguy — APPROVED** [review]

File: jinja/index.html [hunk 3]
```diff
@@ -57,17 +57,17 @@ <h5>Demos <i class="fas fa-angle-double-right"></i></h5>
...
+                                <div class="white-text d-flex justify-content-center">
+                                    <h5>Go to Codebook<i class="fas fa-angle-double-right"></i></h5>
```
Comment (CatalinaAlbornoz) [inline jinja/index.html:70]:
```suggestion
                                    <h5>Codebook<i class="fas fa-angle-double-right"></i></h5>
```

**Review by CatalinaAlbornoz — COMMENTED** [review]
Great idea Álvaro. I only have one small comment to make the wording consistent with the other 2 cards but otherwise it's good to go after committing Nathan's suggestions.

**Review by co9olguy — APPROVED** [review]

**Pull request merged**